lib/actions/sendTemplate.js

```diff
@@ -121,7 +114,7 @@ function getTemplateModel(cfg) {
 function getMetaModel(cfg) {
     const client = mandrill.createClient(cfg);
     const templateName = cfg.templateName;
-    console.log('About to retrieve info about template:', templateName);
+    logger.info('About to retrieve info about template:', templateName);
```

Comment:
Why logger instead of this.logger?

Reply:
It's metadata uploader. No difference here